src/Giraffe/RequestLimitation.fs

```diff
@@ -44,7 +44,7 @@ let mustAcceptAny (mimeTypes: string list) (optionalErrorHandler: OptionalErrorH
             )
 
         match Option.ofObj (headers.Accept :> _ seq) with
-        | Some xs when Seq.map (_.ToString()) xs |> Seq.exists (fun x -> Seq.contains x mimeTypes) -> next ctx
+        | Some xs when Seq.map (_.ToString()) xs |> Seq.exists (fun x -> List.contains x mimeTypes) -> next ctx
```

Comment:
Nitpick, you can probably move the `ToString` inside the Seq.exists.
And I wonder if `Sets` should be used here. The order of mimeTypes doesn't matter here.